src/shared/components/AnimatedError/AnimatedError.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import Lottie from 'react-lottie-player'
+
+import errorAnimation from '../../assets/animations/error.json'
+
+export const AnimatedError: React.FC = () => {
```

Comment:
we could also pass `className` here, to make it stylable. Could be useful for positioning for example